specs/os_version.table

```diff
@@ -19,6 +19,7 @@ extended_schema(DARWIN, [
 
 extended_schema(WINDOWS, [
     Column("install_date", BIGINT, "The install date of the OS."),
+    Column("revision", INTEGER, "Update Build Revision, refers to the specific revision number of a Windows update"),
```

Comment:
This should be in an `extended_schema` block

Reply:
@directionless 
please recheck from your side, it is in the extended_schema block